tests/virt_autotest/install_package.pm

```diff
@@ -99,11 +99,22 @@ sub install_package {
 
     virt_autotest::utils::install_default_packages();
 
-    ###Install required package for window guest installation on xen host
+    #Install required package for window guest installation on xen host
     if (get_var('GUEST_LIST', '') =~ /^win-.*/ && (is_xen_host)) { zypper_call '--no-refresh --no-gpg-checks in mkisofs' }
 
     #Subscribing packagehub from SLE 15-SP4 onwards that enables access to many useful software tools
     virt_autotest::utils::subscribe_extensions_and_modules(reg_exts => 'PackageHub') if (!get_var('AUTOYAST') and is_sle('>=15-sp4') and !is_s390x);
+
+    #Switch all VM Passwords from installed settings files
+    my $setting_file = "/usr/share/qa/virtautolib/data/settings." . locate_sourcefile;
+    my $qa_password = get_var('QA_PASSWORD');
+    my $vm_password = get_var('VM_PASSWORD');
```

Comment:
We are having more and more password related settings. Will be good that each password can be told from its name. So I'd suggest to rename a little bit, eg `VM_PASSWORD` -> `VIRTAUTO_VM_PASSWORD`, get_var('QA_PASSWORD') -> $testapi::password.